### DISCUSSION TITLE
Cannot replace file content

### DISCUSSION THREAD

**Discussion by 107295472** [BurntSushi/ripgrep#2714] [General] [answered]
rg "using NpgsqlTypes;"  'act_ge_bytearray.cs' -r "ddd2rwer;"

**Comment by BurntSushi** [comment, marked as answer]
What's your question? Please provide a reproducible example that includes inputs, actual output and desired/expected output.

**Reply by 107295472** [reply]
I want to replace using NpgsqlTypes; with empty one, that is, delete it and save the file.

**Reply by 107295472** [reply]
![image](https://github.com/BurntSushi/ripgrep/assets/6436116/58c7c237-e96d-4048-8337-b28be0c119b6)
![image](https://github.com/BurntSushi/ripgrep/assets/6436116/fce7da4c-1c0f-4258-8cb9-4a28d5d608bb)

**Reply by 107295472** [reply]
It’s windows without sed

**Reply by BurntSushi** [reply]
Are you expecting ripgrep to modify your file for you? If so, can you say what is unclear about the docs explicitly saying that ripgrep won't do that? 

https://github.com/BurntSushi/ripgrep/blob/648a65f1976cc3b7eb66425024649d71d5befe1e/crates/core/flags/defs.rs#L5897
